pandas-stubs/io/json/_json.pyi

```diff
@@ -18,6 +21,8 @@ from pandas._typing import (
     StorageOptions,
 )
 
+FrameSeriesStrT = TypeVar("FrameSeriesStrT", bound=Union[Series, DataFrame])
```

Comment:
Two things to investigate:
1. I think (but not sure) that you can use `NDFrameT` from `_typing.py` here and everything will work as expected, without having to create a new `TypeVar`
2. If you have to use this `TypeVar` and you can't  use `NDFrameT`,  then make it private to the module with an underscore and remove `Str` from the name, so it is `_FrameSeriesT` .

Reply:
NDFrameT works, so simplified.